### ISSUE TITLE
generate:module argument inconsistencies

### ISSUE THREAD

**Issue by damienmckenna** [hechoendrupal/DrupalConsole#3016]
I'm going through the effort to finish porting all of Metatag's tags from D7 to D8 and using DrupalConsole to create the boilerplate files. However, I've noticed that some of the command line arguments don't work well:

- There's no way of passing an argument to say 'no' to several options, specifically "Define module as feature", "Do you want to add a composer.json file to your module", "Do you want to generate a unit test class", "Do you want to generate a .module file" and "Do you confirm generation".
- There's no difference in the output if the "--learning" argument is passed.
- There's no way of automatically indicating 'yes' to the "Do you confirm generation?" question without specifying either "--yes" which would (in theory) prevent answering other questions.
- Most of the questions do not have a question mark at the end.
- The "Do you confirm generation?" question should provide a summary of what is to be created.

**Comment by damienmckenna** [comment]
I removed this one as it does actually work, it's just confusing.

> There's no way of automatically indicating 'yes' to the "Do you confirm generation?" question without specifying either "--yes" which would (in theory) prevent answering other questions.

**Comment by jmolivas** [comment]
@damienmckenna 
* There's no way of passing an argument to say 'no' to several options, specifically "Define module as feature", "Do you want to add a composer.json file to your module", "Do you want to generate a unit test class", "Do you want to generate a .module file" and "Do you confirm generation".

You can pass the options as inline add also `--no-interaction` to avoid optional questions. 

```
drupal generate:module \
--module="My custom module" \
--machine-name="my_custom_module" \
--module-path="/modules/custom" \
--description="This is a generated module" \
--core="8.x" \
--package="Custom" \
--no-interaction
```

* There's no difference in the output if the "--learning" argument is passed.
The only module taking advantage of this feature is `generate:controller`

* Most of the questions do not have a question mark at the end.
Nice catch, we can fix that by updating the translation files.

* The "Do you confirm generation?" question should provide a summary of what is to be created.
I do like this recommendation

**Comment by damienmckenna** [comment]
Yes, I know I can pass all of the options like that, but if I want to do some arguments via the command line and others via interaction I cannot control several options.

**Comment by jmolivas** [comment]
@damienmckenna what do you think could be a good approach for this?
